src/shared/modules/discovery/discoveryDuck.js

```diff
@@ -112,9 +116,15 @@ export const discoveryOnStartupEpic = (some$, store) => {
     .ofType(APP_START)
     .map(action => {
       if (!action.url) return action
-      const passedURL = getUrlParamValue('connectURL', action.url)
+      const passedURL =
+        getUrlParamValue('dbms', action.url) ||
+        getUrlParamValue('connectURL', action.url)
+
+      const passedDb = getUrlParamValue('db', action.url)
+
       if (!passedURL || !passedURL.length) return action
       action.forceURL = decodeURIComponent(passedURL[0])
+      action.connectTo = passedDb && passedDb[0]
```

Comment:
Right now db param is only respected if there's also a dbms/connectURL param. Just a note on behaviour

Reply:
The reason I didn't use useDb is because it think the same state already has a useDb. But connectTo is a bad name, so perhaps requestedUseDb is a better name?